src/pybind/cephfs/cephfs.pyx

```diff
@@ -369,7 +369,7 @@ cdef make_ex(ret, msg):
     if ret in errno_to_exception:
         return errno_to_exception[ret](ret, msg)
     else:
-        return Error(msg + ': {} [Errno {:d}]'.format(os.strerror(ret), ret))
+        return OSError(ret, msg)
```

Comment:
This is also similar to what rbd.pyx does,
https://github.com/ceph/ceph/blob/master/src/pybind/rbd/rbd.pyx#L1007